source/package-json.d.ts

```diff
@@ -257,8 +257,26 @@ declare namespace PackageJson {
 		typings?: string;
 	}
 
+	export interface Workspaces {
+		/**
+		The packages blob to use for workspaces.
+		*/
+		packages: string[];
+		/**
+		A list of glob patterns for packages that should not be hoisted.
+		*/
+		nohoist: string[];
+	}
+
 	export interface YarnConfiguration {
 		/**
+    Used to configure yarn (and other) workspaces.
```

Comment:
Use tab-indentation.

Would also be useful to link to the Yarn docs about this property.